From 36b339ed18719b3267daeac8c69c2162f610476a Mon Sep 17 00:00:00 2001
From: Praveen Diwakar <praveen.diwakar@intel.com>
Date: Wed, 5 Nov 2014 21:19:23 +0530
Subject: [PATCH] ASoC: Intel: Disable soc fw debug logs

soc fw debug logs are enabled by default. Disabling that
to avoid boot latency

Issue: GMINL-7912

Change-Id: I7e10b0441b903ac1534090460ae47a260256cb77
Signed-off-by: Praveen Diwakar <praveen.diwakar@intel.com>
---
 sound/soc/intel/platform-libs/controls_v2_dpcm.c | 4 ++--
 sound/soc/soc-fw.c                               | 2 --
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/sound/soc/intel/platform-libs/controls_v2_dpcm.c b/sound/soc/intel/platform-libs/controls_v2_dpcm.c
index 5e260b9..9529ccc 100644
--- a/sound/soc/intel/platform-libs/controls_v2_dpcm.c
+++ b/sound/soc/intel/platform-libs/controls_v2_dpcm.c
@@ -2441,13 +2441,13 @@ static int sst_copy_gain_control(struct snd_soc_platform *platform,
 	sm->pvt_data_len = sizeof(*mc_pvt);
 	return 0;
 }
-int sst_fw_kcontrol_find_io(struct snd_soc_platform *platform,
+static int sst_fw_kcontrol_find_io(struct snd_soc_platform *platform,
 		u32 io_type, const struct snd_soc_fw_kcontrol_ops *ops,
 		int num_ops, unsigned long sm, unsigned long mc)
 {
 	int i;
 
-	pr_info("number of ops = %d %x io_type\n", num_ops, io_type);
+	pr_debug("number of ops = %d %x io_type\n", num_ops, io_type);
 	for (i = 0; i < num_ops; i++) {
 		if ((SOC_CONTROL_GET_ID_PUT(ops[i].id) ==
 			SOC_CONTROL_GET_ID_PUT(io_type) && ops[i].put)
diff --git a/sound/soc/soc-fw.c b/sound/soc/soc-fw.c
index 13a1546..f3a3e49 100644
--- a/sound/soc/soc-fw.c
+++ b/sound/soc/soc-fw.c
@@ -17,8 +17,6 @@
  *  data is passed to component drivers for bespoke handling.
  */
 
-#define DEBUG
-
 #include <linux/kernel.h>
 #include <linux/export.h>
 #include <linux/list.h>
-- 
1.9.1

